openfoodfacts_exports/exports/csv_export/csv_from_parquet.py

```diff
@@ -0,0 +1,1045 @@
+            .map_elements(
+                lambda x: process_lang(tags=x, lang=lang),
+                return_dtype=pl.String,
+            ),
+            pl.col("traces_tags")
+            .alias("traces")
+            .map_elements(process_tags, return_dtype=pl.String),
+            pl.col("traces_tags").map_elements(process_tags, return_dtype=pl.String),
+            pl.col("traces_tags")
+            .alias(f"traces_{lang}")
+            .map_elements(
+                lambda x: process_lang(tags=x, lang=lang),
+                return_dtype=pl.String,
+            ),
+            pl.col("serving_size"),
+            pl.col("serving_quantity"),
+            pl.col("no_nutrition_data"),
+            pl.col("additives_n"),
+            pl.col("additives_tags").map_elements(process_tags, return_dtype=pl.String),
+            pl.col("additives_tags")
+            .alias(f"additives_{lang}")
+            .map_elements(
+                lambda x: process_lang(tags=x, lang=lang),
+                return_dtype=pl.String,
+            ),
+            pl.col("nutriscore_score"),
+            pl.col("nutriscore_grade"),
+            pl.col("nova_group"),
+            pl.col("food_groups_tags")
+            .alias("food_groups")
+            .map_elements(process_tags, return_dtype=pl.String),
+            pl.col("food_groups_tags").map_elements(
+                process_tags, return_dtype=pl.String
+            ),
+            pl.col("food_groups_tags")
+            .alias(f"food_groups_{lang}")
+            .map_elements(
+                lambda x: process_lang(tags=x, lang=lang),
+                return_dtype=pl.String,
+            ),
+            pl.col("states_tags")
+            .alias("states")
+            .map_elements(process_tags, return_dtype=pl.String),
+            pl.col("states_tags").map_elements(process_tags, return_dtype=pl.String),
+            pl.col("states_tags")
+            .alias(f"states_{lang}")
+            .map_elements(
+                lambda x: process_lang(tags=x, lang=lang),
+                return_dtype=pl.String,
+            ),
+            pl.col("ecoscore_score"),
+            pl.col("ecoscore_grade"),
+            pl.col("nutrient_levels_tags").map_elements(
+                process_tags, return_dtype=pl.String
+            ),
+            pl.col("product_quantity"),
+            pl.col("owner"),
+            pl.col("data_quality_errors_tags").map_elements(
+                process_tags, return_dtype=pl.String
+            ),
+            pl.col("unique_scans_n").cast(pl.Int32),
+            pl.col("popularity_tags").map_elements(
+                process_tags, return_dtype=pl.String
+            ),
+            pl.col("completeness"),
+            pl.col("last_image_t"),
+            pl.col("last_image_t")
+            .alias("last_image_datetime")
+            .map_elements(datetime.fromtimestamp, return_dtype=pl.Datetime),
+            # NOTE: Unclear what is the main category
```

Comment:
This is the lowest level category.
See: https://github.com/openfoodfacts/openfoodfacts-server/blob/4f3149ee045e19503acf7db265f18a515621c677/scripts/export_database.pl#L412

I guess this information can be important for some studies, but I don't have clear use cases in my mind. 

I have checked on Mirabelle and I can tell that it is always the last item of `categories_tags`.

It's a bit redundant data (such as created_t and created_datetime), but I would be in favor to keep it for backward compatibility.